doc/application/index.rst

```diff
@@ -1446,6 +1446,162 @@ After running the preprocessor, the final device tree used in the
 build is created by running the device tree compiler, ``dtc``, on the
 preprocessor output.
 
+Building and Configuring multiple images
+****************************************
+
+When firmware update and bootloaders are involved in an embedded
+application the final firmware that is flashed onto the device will
+usually need to consist of not one executable, but multiple
```

Comment:
You seem to use "image" and "executable" interchangeably. If they are actually interchangeable than settle on just one. If they're not is the difference explained somewhere?

Reply:
I'm using executable here because I believe it is the word that most people will understand. And then I switch to using 'image' in the next paragraph after explaining that they are the same thing.